### PR TITLE
reintroduce netstandard2.0 as a target

### LINKED ISSUE #111: The latest version no longer supports .NET Core 2.1
**Describe the bug**
I'm using Snapshooter for some tests in a project that targets .NET Core 2.1 and 3.1. In the latest 0.5.7 release of Snapshooter.Xunit .NET Core 2.1 is no longer supported.

This change was made in [Snapshooter for NUnit tests. (#110)](https://github.com/SwissLife-OSS/snapshooter/commit/60b9158b8941b940c6e01e6ca7862b69dfb9409e). The target framework was changed from `netstandard2.0` to `netstandard2.1` which isn't supported when targeting `netcoreapp2.1`.

It was probably an intentional change , but I wanted to check as it was just a patch version bump.

**To Reproduce**
Failing tests can be viewed here https://github.com/CycloneDX/cyclonedx-dotnet/pull/247
You can also clone that repo locally if needed. `dotnet test` will run tests for 2.1 and 3.1. Although you need .NET Core 2.1 and 3.1 installed to run the tests. Although there is a devcontainer configuration. Repro by updating the Snapshooter.Xunit package references.

**Expected behavior**
Continued support for .NET standard 2.0 or major version bumps for breaking changes (with the caveat that right now this is still on 0.x.x versioning, so I can understand breaking changes coming in like this until a 1.0.0 release).

**Desktop (please complete the following information):**
 - OS: Linux
 - Version Ubuntu 20.04

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by RohrerF**
Summary of the changes

- I added ```netstandard2.0``` as a target for all library projects

Resolves #111

### CODE REVIEW

**Comment by coderpatros** [comment]
Out of interest, why target multiple .NET standard versions? Anything that can use .NET standard 2.1 can use 2.0.

File: src/Snapshooter.Json/Snapshooter.Json.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>netstandard2.0; netstandard2.1</TargetFrameworks>
```
Comment (nscheibe) [inline src/Snapshooter.Json/Snapshooter.Json.csproj:4]: netstandard2.0 would be enough. However, thats okay for me.

**Review by nscheibe — DISMISSED** [review]
Nice!

File: src/Snapshooter.Json/Snapshooter.Json.csproj (hunk 1 above)
Comment (RohrerF) [reply src/Snapshooter.Json/Snapshooter.Json.csproj:4]: i willl change it to netstandard2.0

Comment (nscheibe) [reply src/Snapshooter.Json/Snapshooter.Json.csproj:4]: Perfect, thx

**Review by nscheibe — APPROVED** [review]
Perfect!

**Pull request merged**